qemu/tests/passt_stability_tests.py

```diff
@@ -0,0 +1,142 @@
+import os
+import re
+import shutil
+import time
+
+from aexpect import ShellCmdError
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import utils_net
+from virttest import utils_misc
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    passt stability tests
+
+    1) Boot up VM with passt backend
+    2) Prepare the iperf environment
+    3) Select host to start iperf server, guest as client
+    4) Execute iperf tests, analyze the result
+    5) Finish test and cleanup host environment
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+    def transfer_iperf_package():
+        receive_cmd = params.get("receive_cmd")
+        serial_session.sendline(receive_cmd)
+        time.sleep(5)
+        sent_cmd = params.get("sent_cmd")
+        process.run(sent_cmd % host_iperf_src_path)
+
+    def iperf_compile(src_path, dst_path, serial_session=None):
+        """Compile iperf and return its binary file path."""
+        iperf_version = params["iperf_version"]
+        iperf_source_path = os.path.join(dst_path, iperf_version)
+        compile_cmd = params["linux_compile_cmd"] % (src_path, dst_path,
+                                                     iperf_source_path)
+        try:
+            if serial_session:
+                test.log.info("Compiling %s in guest...", iperf_version)
+                serial_session.sendline(compile_cmd)
+            else:
+                test.log.info("Compiling %s in host...", iperf_version)
+                process.run(compile_cmd, shell=True, verbose=False)
+        except (process.CmdError, ShellCmdError) as err_msg:
+            test.log.error(err_msg)
+            test.error("Failed to compile iperf")
+        else:
+            iperf_bin_name = re.sub(r'[-2]', '', iperf_version.split('.')[0])
+            return os.path.join(iperf_source_path, 'src', iperf_bin_name)
+
+    def iperf_server_start():
+        """Start iperf server"""
+        info_text = "Start iperf server session in %s with cmd: %s"
+        iperf_server_cmd = iperf_server_options % host_iperf_bin
+        try:
+            test.log.info(info_text, "host", iperf_server_cmd)
+            server_output = process.system_output(iperf_server_cmd, timeout=300, verbose=False).decode()
+        except Exception as err_msg:
+            test.log.error(str(err_msg))
+            test.error("Failed to start iperf session")
+        else:
+            test.log.debug("Full connection server log:\n%s", server_output)
+            print(server_output)
```

Comment:
Well, I don't think we need the print func.

Reply:
Sure, I have delete this line, please help review it again when you have free time. Thanks a lot.